sklearn/impute.py

```diff
@@ -917,7 +931,24 @@ def fit_transform(self, X, y=None):
             if self.verbose > 0:
                 print('[IterativeImputer] Ending imputation round '
                       '%d/%d, elapsed time %0.2f'
-                      % (i_rnd + 1, self.n_iter, time() - start_t))
+                      % (i_rnd + 1, self.max_iter, time() - start_t))
+
+            # stop early if difference between consecutive imputations goes up.
+            # if so, back off to previous imputation
+            if not self.sample_posterior:
+                norm_diff = np.linalg.norm(Xt - Xt_previous) \
```

Comment:
Use parentheses rather than `\` for line continuation.

Is this an obvious definition of "the difference between imputations"?